digdag-docs/src/operators/py.md

```diff
@@ -76,6 +76,112 @@ See [Python API documents](../python_api.html) for details including variable ma
     print("simple execution")
   ```
 
+  You can pass arguments to class for initialization using `_export` as the following:
+
+  ```yaml
+  # sample.dig
+  +some_task:
+    _export:
+      required1_1: awesome execution
+      required1_2: "awesome execution"
+      required2: {a: "a"}
+      required3: 1
+      required4: 1.0
+      required5: [a, 1, 1.0, "a"]
+    py>: tasks.MyWorkflow.my_task
+  ```
+
+  Also, you can do the same thing by defining arguments under the `py>:` operation as the following:
```

Comment:
Could you exchange the order of description ?
I think pass by argument of `py>` is default way. `_export` is another way.
So this block should be described at first. Then describe L79.

Reply:
Thank you for your review. I changed the order, please check the modification.